float64  trailer1
float64  trailer2
float64  trailer3